inc/saiacl.h

```diff
@@ -2623,10 +2673,28 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_FIELD_TUNNEL_TERMINATED = SAI_ACL_ENTRY_ATTR_FIELD_START + 0x15f,
 
+    /**
+     * @brief SRC BINCODE
+     *
+     * @type sai_acl_field_data_t sai_uint32_t
```

Comment:
To make reference counting easier, will it be better to model this as:  * @type sai_acl_field_data_t sai_object_id_t of type sai_prefix_compression_entry

Reply:
I don't think this needs to be an entry, this is a bincode(META) value that we are sending in to ACL to match on. A prefix compression entry is the actual key that would be used. We want to match on the result of the key. So we should match on either BINCODE or META if we decide to go with that name.

The reason we are not passing in the entry structure here, because the BINCODE or META is not part of that entry. This is similar to sairoute.cpp. in SAI route we have a route entry and the result of that entry is the next hop. 

if this is still confusing we can discuss it next time we have a meeting about this or let me know and I can try to give a better explanation.

Reply:
@rck-innovium Can you please resolve this since we are not going to make this change as discussed in today's meeting